modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -1096,4 +1096,29 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-171627. Verify that that on Target Market, just 1 ~ 3 options can be selected "
+	@priority = 4
+	test ViewTargetMarketOptionsError {
+		property solutions.acceptance = "true";
+		property test.name.skip.portal.instance = "PRMMDFRequest#OnlyManagersCanSeeStateManager";
```

Comment:
Hi @pedroamarall, please replace the test name from `OnlyManagersCanSeeStateManager` to `ViewTargetMarketOptionsError`